src/main/java/sirius/db/jdbc/StatementCompiler.java

```diff
@@ -65,9 +68,11 @@ protected PreparedStatement getStmt() throws SQLException {
      * @param context the context defining the parameters available
      */
     protected void buildParameterizedStatement(String query, Context context) throws SQLException {
-        List<Object> params = Lists.newArrayList();
+        this.params = Lists.newArrayList();
```

Comment:
`newArrayList()` is deprecated since Java 1.7
(https://github.com/google/guava/blob/master/guava/src/com/google/common/collect/Lists.java#L138)